Cargo.toml

```diff
@@ -110,3 +110,6 @@ crate-type = ["bin"]
 [[example]]
 name = "callback"
 crate-type = ["bin"]
+
+[patch.crates-io]
+wasmparser =  { git = "https://github.com/AdamSLevy/wasmparser", branch = "OperatorClonev0.39.3" }
```

Comment:
Can you create a PR into `wasmparser` so we don't need to patch it here?

https://github.com/bytecodealliance/wasmparser/compare/master...AdamSLevy:OperatorClonev0.39.3

Reply:
https://github.com/bytecodealliance/wasmparser/pull/157

Reply:
The issue you may find with my PR is that it is based on the latest wasmparser version. This version needs to match the version given by some external dependencies as well. That locks us to 0.39.3 atm, hence my branch based on that version specifically.

Reply:
This is now merged upstream.